### PR TITLE
refactor(root): Rename Echo to Bridge

### PR DESCRIPTION
**Pull request by rifont**
### What changed? Why was the change needed?

* Rename Echo to Bridge

Related Enterprise PR: https://github.com/novuhq/packages-enterprise/pull/161

### Screenshots

_Sanity Testing in Local Studio, execution is successful_
image

_Sanity testing old workflows in Dev_
image

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/web/src/components/layout/components/BridgeStatus.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 import { useDisclosure } from '@mantine/hooks';
-import { IS_DOCKER_HOSTED } from '../../../config';
+import { IS_DOCKER_HOSTED } from '../../../config/index';
 
 export function EchoStatus() {
   const [opened, { close, open }] = useDisclosure(false);
 
   const { environment } = useEnvironment();
-  const echoEnabled = !!environment?.echo?.url && !IS_DOCKER_HOSTED;
+  const isBridgeEnabled = !!environment?.echo?.url && !IS_DOCKER_HOSTED;
```
Comment (rifont) [inline apps/web/src/components/layout/components/BridgeStatus.tsx:13]: We will have a follow-up PR to start setting `environment.bridge.url`, then we'll run a manual migration against MongoDB to backfill the value everywhere, then we will kill `environment.echo.url` for good.

**Review by SokratisVidros — APPROVED** [review]
👋 👋

File: enterprise/packages/bridge-api/src [hunk 2]
```diff
@@ -0,0 +1 @@
+../../../.source/bridge-api/src
```
Comment (scopsy) [inline enterprise/packages/bridge-api/src:1]: What does this file do?

**Review by scopsy — APPROVED** [review]

File: enterprise/packages/bridge-api/src (hunk 2 above)
Comment (rifont) [reply enterprise/packages/bridge-api/src:1]: This is the file holding the symlink from Public `enterprise/packages/<lib>/src` to Enterprise `.source/<lib>/src`

**Pull request merged**